keras/utils/data_utils.py

```diff
@@ -477,26 +493,43 @@ def start(self, workers=1, max_queue_size=10):
             self.executor = multiprocessing.Pool(workers)
         else:
             self.executor = ThreadPool(workers)
+        self.workers = workers
         self.queue = queue.Queue(max_queue_size)
         self.stop_signal = threading.Event()
         self.run_thread = threading.Thread(target=self._run)
         self.run_thread.daemon = True
         self.run_thread.start()
 
+    def wait_queue(self):
```

Comment:
Does this need to be a public method, or should it be private? Doesn't seem to be used outside of this class.